CHANGELOG.md

```diff
@@ -1,3 +1,30 @@
+# Change Log
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/)
+and this project adheres to [Haskell Package Versioning Policy](https://pvp.haskell.org/).
```

Comment:
I agree with the PVP for my code.  But the current wording of the PVP also contains this wording:

> Note that modifying imports or depending on a newer version of another package may cause extra orphan instances to be exported and thus force a major version change.

Some people interpret this as: In theory one of your dependencies (e.g. `directory`) could add an orphan instance, for that reason following the PVP means you have to specify upper bounds for all your dependencies.  I'm not going to do that.